doc/tutorial/alphabet/index.md

```diff
@@ -0,0 +1,329 @@
+\endassignment
+
+<br/>
+
+---
+
+# The nucleotide alphabets
+
+Nucleotides are the components of (Deoxy)Ribonucleic acid (DNA/RNA) and contain one of the nucleobases
+Adenine (A), Cytosine (C), Guanine (G), Thymine (T, only DNA) and Uracil (U, only RNA).
+In SeqAn the alphabets seqan3::dna4 and seqan3::rna4 contain exactly the four respective nucleotides.
+The trailed number in the alphabets' name represents the number of entities the alphabet holds –
+we denote this number as *alphabet size*.
+For instance, the alphabet seqan3::dna5 represents five entities as it contains the additional symbol 'N'
+to refer to an unknown nucleotide.
+
+<br/>
+
+## Creation of alphabet symbols
+Let's look at some example code, which demonstrates how characters of the seqan3::dna4 alphabet are created.
+\snippet alphabet_main.cpp create
+\snippet alphabet_main.cpp closing
+
+We have shown three solutions for assigning variables of alphabet type.
+1. Assignment by character literal, i.e. appending the operator `_dna4` to the respective char symbol. <br/>
+   This is the handiest way, as it can be used temporarily (without creating a variable) as well.
+2. Assignment by `char` via the global function seqan3::assign_char. <br/>
+   This is useful if the assignment target already exists, e.g. in a sequence vector.
+3. Assignment by rank via the global function seqan3::assign_rank. <br/>
+   May be used when the *rank* is known.
+
+<br/>
+
+## The rank of an alphabet symbol
+The rank of a symbol is a number in range \[0..alphabet_size), where each number is paired with
+an alphabet symbol by a bijective function. For instance in seqan3::dna4 the bijection is
+<br/> `A ⟼ 0` <br/> `C ⟼ 1` <br/> `G ⟼ 2` <br/> `T ⟼ 3`.
+
+SeqAn provides the function seqan3::to_rank for converting a symbol to its rank value,
+as demonstrated in the following code example. Note that the data type of the rank is usually the smallest possible
+unsigned type that is required for storing the values of the alphabet.
+\snippet alphabet_main.cpp rank
+
+<br/>
+
+## The char representation of an alphabet symbol
+
+Our alphabets also have a character representation, because it is more intuitive to work
+with them than using the rank. Each alphabet symbol is represented by its respective character
+whenever possible (`A ⟼ 'A'`). Analogously to the rank, SeqAn provides the function 
+seqan3::to_char for converting a symbol to its character representation.
+
+\snippet alphabet_main.cpp char
+
+Above you have seen that you can assign an alphabet symbol from a character with seqan3::from_char. 
+In contrast to the rank interface, this assignment is not a bijection, because the whole spectrum of available
+chars is mapped to values inside the alphabet. For instance, assigning to seqan3::dna4 from any character other
+than `C`, `G` or `T` results in the value 'A'_dna4 and assigning from any character except `A`, `C`, `G` or `T`
```

Comment:
```suggestion
than `C`, `G` or `T` results in the value `'A'_dna4` and assigning from any character except `A`, `C`, `G` or `T`
```

Reply:
I tried before, but it doesn't display properly :(
It transforms all the following text in typewriter, because the first '`' is somehow ignored.

Reply:
weird... but oh well then leave it